### PR TITLE
Copy content from existing course version fixed

### PR DESCRIPTION
**Pull request by a22ahmom**
It seemed like the solution to the problem was to add another else if statement but for CPYVRS and fix a syntax error. So now it should be possible to copy content from existing course version when creating a new course version. The correct format for creating a new course version is

Version ID: 2001
Version Name: HT01
Start Date: 01/04/2024
End Date: 31/05/2024
MOTD: MOTD
Copy content from: ST20 - 52432

### CODE REVIEW

**Comment by d22mikwe** [comment]
To Tester:
Check that copying of courses works.
Visual problems and copied content of duggor will be dealt with in a separate issue.

**Review by a22maxra — APPROVED** [review]
image
You are now able to create new versions and a course that gets saved

File: Shared/dugga.js [hunk 1]
```diff
@@ -1015,17 +1015,50 @@ function AJAXService(opt,apara,kind)
...
-		}else if(kind=="VARIANTPDUGGA"){
+        }
+		/*$.ajax({
+			url : "courseedservice.php",
+			type: "POST",
+			data: "opt="+opt+para,
+			dataType: "json",
+			success: returnedCourse
+		});*/
+	}else if(kind=="VARIANTPDUGGA"){
```
Comment (d22mikwe) [inline Shared/dugga.js:1061]: Remove comment?

**Review by d22mikwe — CHANGES_REQUESTED** [review]

**Review by d22mikwe — APPROVED** [review]

**Comment by d22mikwe** [comment]
LGTM!

**Pull request merged**